openapi doesn't work with sphinx 1.6.1

```
$ make html
sphinx-build -b html -d build/doctrees   source build/html
Running Sphinx v1.6.1
making output directory...
loading translations [ja]... 
/usr/local/lib/python3.6/site-packages/sphinxcontrib/openapi.py:23: RemovedInSphinx17Warning: sphinx.util.compat.Directive is deprecated and will be removed in Sphinx 1.7, please use the standard library version instead.
  from sphinx.util.compat import Directive

Exception occurred:
  File "/usr/local/lib/python3.6/site-packages/sphinxcontrib/openapi.py", line 195, in setup
    if 'http' not in app.domains:
AttributeError: 'Sphinx' object has no attribute 'domains'
The full traceback has been saved in /tmp/sphinx-err-f5_cfbju.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
Makefile:55: recipe for target 'html' failed
make: *** [html] Error 1
```

I think https://github.com/sphinx-doc/sphinx/issues/3760 is same issue.

--- a/CHANGES
+++ b/CHANGES
@@ -6,6 +6,12 @@ Here you can see the list of changes between each 'sphinxcontrib-openapi'
 release.
 
 
+0.3.1 (2017-05-19)
+==================
+
+- Make 'sphinxcontrib-openapi' compatible with Sphinx 1.6.
+
+
 0.3.0 (2017-01-29)
 ==================
 

--- a/sphinxcontrib/openapi.py
+++ b/sphinxcontrib/openapi.py
@@ -17,13 +17,10 @@ import yaml
 import jsonschema
 
 from docutils import nodes
-from docutils.parsers.rst import directives
+from docutils.parsers.rst import Directive, directives
 from docutils.statemachine import ViewList
 
-from sphinx.util.compat import Directive
 from sphinx.util.nodes import nested_parse_with_titles
-
-from sphinxcontrib import httpdomain
 
 
 # Dictionaries do not guarantee to preserve the keys order so when we load
@@ -214,6 +211,5 @@ class OpenApi(Directive):
 
 
 def setup(app):
-    if 'http' not in app.domains:
-        httpdomain.setup(app)
+    app.setup_extension('sphinxcontrib.httpdomain')
     app.add_directive('openapi', OpenApi)

